infrastructure/config/config.go

```diff
@@ -122,6 +122,7 @@ type Flags struct {
 	IsArchivalNode                  bool          `long:"archival" description:"Run as an archival node: don't delete old block data when moving the pruning point (Warning: heavy disk usage)'"`
 	AllowSubmitBlockWhenNotSynced   bool          `long:"allow-submit-block-when-not-synced" hidden:"true" description:"Allow the node to accept blocks from RPC while not synced (this flag is mainly used for testing)"`
 	EnableSanityCheckPruningUTXOSet bool          `long:"enable-sanity-check-pruning-utxo" hidden:"true" description:"When moving the pruning point - check that the utxo set matches the utxo commitment"`
+	AllowSubmittingNonDAABlocks     bool          `long:"allow-non-daa-blocks" hidden:"false" description:"When submitting blocks via RPC - accept blocks which are out of virtual daa window (although such blocks are not rewarded anyhow)"`
```

Comment:
@someone235  is `hidden:"false"` needed here ?

Reply:
I think it's better to move this flag directly to the submitBlock RPC command